projects/admin/src/app/scss/styles.scss

```diff
@@ -270,3 +270,7 @@ json-schema-form {
   background-color: #f8f9fa;
   border-radius: 5px;
 }
+
+.user-id-modal-width {
+  max-width: 1000px;
+}
```

Comment:
should be unecessary if your use `modal-lg`

Reply:
Right, I got some problem when I tried the first time, but now it works! Thanks.